app/src/lang/translations/en-US.yaml

```diff
@@ -2406,6 +2406,7 @@ triggers:
   webhook:
     name: Webhook
     description: Triggers on an incoming HTTP request
+    error_on_reject: Error on Reject
```

Comment:
As we re-use (see [1](https://github.com/directus/directus/blob/67c7fa137e699ce201d9f3e0cde1ab84e64a358d/app/src/modules/settings/routes/flows/triggers.ts#L232) and [2](https://github.com/directus/directus/blob/67c7fa137e699ce201d9f3e0cde1ab84e64a358d/app/src/modules/settings/routes/flows/triggers.ts#L372)) our [async translation](https://github.com/directus/directus/blob/67c7fa137e699ce201d9f3e0cde1ab84e64a358d/app/src/lang/translations/en-US.yaml#L2410) due to duplicate wording, we should do the same for [error_on_reject](https://github.com/directus/directus/blob/67c7fa137e699ce201d9f3e0cde1ab84e64a358d/app/src/lang/translations/en-US.yaml#L2422) instead of introducing a duplicate wording in another location.

Alternatively we can move this under `common`? Thoughts?